ONSdigital/dp-data-api
# Update app to use log.go v3

### What

Switch app to use v3 of the dp.log library. 

⚠️ This library is unreleased so this draft PR uses a specific commit rather than a proper version number. The library needs to be released and this branch updated with the correct version before this PR is merged.

### How to review

- Run the app and ensure logging still works as expected. 
- Ensure all tests still pass

### Who can review

Anyone
